docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,191 @@
+
+### Server OS
+
+Although AvalancheGo can run on a MacOS or a Windows computer, we strongly recommend running nodes on computers running Linux as they are designed specifically for server loads and all the tools and utilities needed for administering a server are native to Linux.
+
+### Hardware Specification
+
+For running AvalancheGo as a validator on the Primary Network the recommended configuration is as follows:
+
+* CPU: Equivalent of 8 AWS vCPU
+* RAM: 16 GiB
+* Storage: 1TB with at least 3000 IOPS
+* OS: Ubuntu 18.04/20.04
+* Network: Reliable IPv4 or IPv6 network connection, with an open public port
+
+That is the configuration sufficient for running a Primary Network node. Any resource requirements for your Subnet come on top of this, so you should not go below this configuration, but may need to step up the specification if you expect your Subnet to handle a significant amount of transactions.
+
+Be sure to set up monitoring of resource consumption for your nodes because resource exhaustion may cause your node to slow down or even halt, which may severely impact your Subnet negatively.
+
+### Server Location
+
+You can run a node on a physical computer that you own and run, or on a cloud instance. Although running on your own HW may seem like a good idea, unless you have a sizeable DevOps 24/7 staff we recommend using cloud service providers as they generally provide reliable computing resources that you can count on to be properly maintained and monitored.
+
+#### Local Servers
+
+If you plan on running nodes on your own hardware, make sure they satisfy the minimum HW specification as outlined earlier. Pay close attention to proper networking setup, making sure the p2p port (9651) is accessible and public IP properly configured on the node. Make sure the node is connected to the network physically (not over WiFi), and that the router is powerful enough to handle a couple of thousands of persistent TCP connections and that network bandwidth can accommodate at least 5Mbps of steady upstream and downstream network traffic.
+
+When installing the AvalancheGo node on the machines, unless you have a dedicated DevOps staff that will take care of node setup and configuration, we recommend using the [installer script](../nodes/build/set-up-node-with-installer.md) to set up the nodes. It will abstract most of the setup process for you, set up the node as a system service and will enable easy node upgrades.
+
+#### Cloud Providers
+
+There are a number of different cloud providers. We have documents that show how to set up a node on the most popular ones:
+* [Amazon Web Services](../nodes/build/setting-up-an-avalanche-node-with-amazon-web-services-aws.md)
```

Comment:
[MD032/blanks-around-lists: Lists should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines)